src/constants.h

```diff
@@ -215,6 +215,9 @@ constexpr const char* SUMO_MULTIHOP =
 constexpr const char* SUMO_ALWAYS_ON_ANDROID =
     "https://support.mozilla.org/kb/how-enable-always-vpn-android";
 
+constexpr const char* SUMO_ALLOW_BACKGROUND_MACOS =
+    "https://support.mozilla.org/en-US/kb/allow-vpn-run-background-macos";
```

Comment:
```suggestion
    "https://support.mozilla.org/kb/allow-vpn-run-background-macos";
```

Let SUMO handle localization decisions.